x-pack/plugins/session_view/public/components/detail_panel_host_tab/index.tsx

```diff
@@ -5,194 +5,411 @@
+                  </DetailPanelCopy>
+                ),
+              },
+              {
+                title: <DetailPanelListItem>os.full</DetailPanelListItem>,
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`host.os.full: "${hostData.os.full}"`}
+                    tooltipContent={hostData.os.full}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {hostData.os.full}
+                    </EuiTextColor>
+                  </DetailPanelCopy>
+                ),
+              },
+              {
+                title: <DetailPanelListItem>os.kernel</DetailPanelListItem>,
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`host.os.kernel: "${hostData.os.kernel}"`}
+                    tooltipContent={hostData.os.kernel}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {hostData.os.kernel}
+                    </EuiTextColor>
+                  </DetailPanelCopy>
+                ),
+              },
+              {
+                title: <DetailPanelListItem>os.name</DetailPanelListItem>,
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`host.os.name: "${hostData.os.name}"`}
+                    tooltipContent={hostData.os.name}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {hostData.os.name}
+                    </EuiTextColor>
+                  </DetailPanelCopy>
+                ),
+              },
+              {
+                title: <DetailPanelListItem>os.platform</DetailPanelListItem>,
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`host.os.platform: "${hostData.os.platform}"`}
+                    tooltipContent={hostData.os.platform}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {hostData.os.platform}
+                    </EuiTextColor>
+                  </DetailPanelCopy>
+                ),
+              },
+              {
+                title: <DetailPanelListItem>os.version</DetailPanelListItem>,
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`host.os.version: "${hostData.os.version}"`}
+                    tooltipContent={hostData.os.version}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {hostData.os.version}
+                    </EuiTextColor>
+                  </DetailPanelCopy>
+                ),
+              },
+            ]}
+          />
+        </EuiPanel>
+      </EuiAccordion>
+      {processOrchestrator && processContainer && (
```

Comment:
I dont think you need to check `processOrchestrator` and `processContainer` here since you should be able to render all the data using `containerData` and `orchestratorData`.

Unless it's by design that we don't want to show the accordions when there are no data.

Reply:
it's by design, in the case the session came from a non-cloud environment, we don't want to show Container or Orchestrator (as its gonna be empty)

Reply:
Sounds good. In that case should we separate render logic for orchestrator and container? Right now both of them will not show if just one of them doesn't have data.

Reply:
And in the case of them not existing, we won't have to call `getContainerData` or `getOrchestratorData`. We can update the useMemos above to only call those functions when needed.